controllers/replication.storage/volumereplication_controller.go

```diff
@@ -380,6 +380,17 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			if ts != nil {
 				lastSyncTime := metav1.NewTime(ts.AsTime())
 				instance.Status.LastSyncTime = &lastSyncTime
+				instance.Status.LastSyncDuration = nil
+				instance.Status.LastSyncBytes = nil
```

Comment:
i dont think you should make this nil. You should preserve the last data because you already have lastSyncTime, Duration, Byte to give to complete sync information.

Reply:
> i dont think you should make this nil. You should preserve the last data because you already have lastSyncTime, Duration, Byte to give to complete sync information.

@Madhu-1 , 
We'll reach this point only when csi driver gives us a valid new lastSyncTime.

It'll be incorrect to keep previous duration and bytes data with new lastSyncTime according to me.

Reply:
okay, just read the code, LGTM